modules/openapi-generator/src/main/resources/typescript/api_doc.mustache

```diff
@@ -22,27 +22,25 @@ Method | HTTP request | Description
 
 
 ```typescript
-import { {{{moduleName}}} } from '{{{projectName}}}';
-import * as fs from 'fs';
+import {{{moduleName}}} from '{{{projectName}}}';
 
 const configuration = {{{moduleName}}}.createConfiguration();
 const apiInstance = new {{{moduleName}}}.{{classname}}(configuration);
 
 {{#hasParams}}
-let body:{{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
+const request: {{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
 {{#allParams}}
-  // {{{dataType}}}{{#description}} | {{{description}}}{{/description}}{{^required}} (optional){{/required}}
```

Comment:
given that we define the request with an explicit type, there is no point adding the dataType into the generated code. Anyone copy & pasting the code into their Typescript project would see what types they can pass.